CHANGELOG.md

```diff
@@ -2,25 +2,65 @@
 
 ## UNRELEASED
 
+### Improvements
+
+* [#1779](https://github.com/crypto-org-chain/cronos/pull/1779) Upgrade rocksdb to `v9.11.2`.
```

Comment:
Is it the only differences between `1.4.6` and `main`?

Why is it not included to `v1.4.x` if it is not a state machine breaking?

Reply:
rocksdb upgrade is not always smooth before, so we are a little bit cautious with that now.